packages/components/src/components/loaders/Spinner/Spinner.stories.tsx

```diff
@@ -1,22 +1,45 @@
 import { Meta, StoryObj } from '@storybook/react';
 import { Spinner as SpinnerComponent, SpinnerProps } from './Spinner';
-import { framePropsStory } from '../../common/frameProps';
 
 const meta: Meta = {
     title: 'Loaders/Spinner',
     component: SpinnerComponent,
 } as Meta;
 export default meta;
 
-export const Spinner: StoryObj<SpinnerProps> = {
+export const Refresh: StoryObj<SpinnerProps> = {
```

Comment:
What about `Default` or something else? Not sure if Refresh is correct. Also I might prefer to have only one story for it with all props, but I don't have strong opinion here, it's up to you,